dev/PushNotifications/PushNotificationReceivedEventArgs.cpp

```diff
@@ -115,33 +115,5 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
             m_backgroundTaskInstance.Canceled(token);
         }
     }
-
-    bool PushNotificationReceivedEventArgs::Handled()
-    {
-        if (!m_unpackagedAppScenario)
-        {
-            THROW_HR_IF_NULL_MSG(E_ILLEGAL_METHOD_CALL, m_args, "Background activation cannot call this.");
-
-            return m_args.Cancel();
-        }
-        else
-        {
-            return m_handledUnpackaged;
```

Comment:
m_handledUnpackaged isn't used anywhere now, please remove it in the header.

Reply:
Yes. Please delete all unised variables, function arguments etc. that are no longer being used.